cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,139 @@
+
+- **Routing** - Once we have clarified any questions we have on a submitted ticket and we are able to replicate the problem internally, the ticket is ready to be routed to the appropriate team at Cypress.
+
+- **Prioritization** - Teams meet regularly to review tickets that have been routed to them via triage. They evaluate each ticket based on the a number of criteria outlined in our [prioritization rubric](#prioritization-rubric). Each dimension of our prioritization rubric carries a weight. The following equation is then used to give us a **relative priority**. The relative priority score is then divided by the estimated effort to address the ticket to give us a **priority-to-effort** score which we then use to determine which tickets are the highest priority. 
+**Important note - These priorities do not dictate when a team will be able to start work. This simply gives teams the data they need to make informed prioritization choices.**
+
+- **Resolution and Verification** - Once an issue has been picked up for work, it will have the label **Stage: Under Development** attached. Developers will then begin work on the issue. Sometimes the scope of the work will be greater than the ticket submitted, and a parent ticket will be created to encapsulate the entire scope of the work. A link to the original ticket will be created in the new ticket. 
+
+- **Release** - Once work has been finished, the ticket will be scheduled for release. If the changes are non-breaking, we generally aim to include the work in the next minor release every-other Tuesday. If the changes made to address the issue require breaking changes, the issue will be scheduled for our next major release roughly every quarter.
+
+### The Power of a Great Ticket
+
+One of the best ways to help get your ticket validated, replicated, routed and prioritized is to include as much information as possible.  Our issue template will walk you through the most common information that we will need to best troubleshoot the problem.  **Please do not open issues from GitHub discussion comments.**  
```

Comment:
Hmm should we/is there any way to also provide the guidance around not opening issues from Github Discussions comments somewhere within Github Discussions? Since this is a feature of Github Discussions I could see this getting unfortunately ignored

Reply:
There is not a way currently to disable that from what I can find.  Unfortunately everything in this document **can** be ignored if a user so chooses.